runtime/cere/src/lib.rs

```diff
@@ -1496,10 +1496,7 @@ parameter_types! {
 	pub DummyPalletAccountId: AccountId = DummyPalletId::get().into_account_truncating();
 }
 /// Runtime migrations
-type Migrations = (
-	pallet_balances::migration::MigrateToTrackInactive<Runtime, DummyPalletAccountId>,
-	pallet_contracts::Migration<Runtime>,
-);
+type Migrations = ();
```

Comment:
Probably we will need to provide a migration that moves existing preimages (if any) from the pallet-democracy's internal storage to the external pallet-preimages storage.

Reply:
I checked storage migrations for democracy / preimages & scheduler pallets and there is no such thing. I believe it's going to be time consuming to migrate un-executed preimage call hashes so i think we can skip this part.